src/ios.ts

```diff
@@ -119,7 +119,16 @@ export const iosApi = <RunTarget extends SupportedRunTarget<'ios'>>(
     },
     clearStuckModals: asyncUnimplemented('clearStuckModals') as never,
 
-    isAppInstalled: asyncUnimplemented('isAppInstalled') as never,
+    isAppInstalled: async (appId) => {
+        const { stdout } = await execa('ideviceinstaller', ['-l', '-o', 'list_all']);
+        return (
+            stdout
+                .split('\n')
+                // The first line is the header.
+                .slice(1)
+                .some((l) => l.startsWith(`${appId},`))
```

Comment:
Why don’t you just `grep`?

Reply:
I prefer not to rely on shell stuff and would rather do things in JS.

Reply:
But `grep` is part of POSIX?

Reply:
What does POSIX have to do with me not wanting to rely on shell stuff for things I can do in JS? :o